src/material/datepicker/datepicker-content.html

```diff
@@ -15,5 +16,16 @@
     (yearSelected)="datepicker._selectYear($event)"
     (monthSelected)="datepicker._selectMonth($event)"
     (viewChanged)="datepicker._viewChanged($event)"
-    (_userSelection)="_handleUserSelection($event)">
-</mat-calendar>
+    (_userSelection)="_handleUserSelection($event)"></mat-calendar>
+
+  <!-- Invisible close button for screen reader users. -->
+  <button
+    type="button"
+    mat-raised-button
+    color="primary"
+    class="mat-datepicker-close-button"
```

Comment:
can this just be .cdk-visually-hidden instead?

Reply:
The `mat-datepicker-close-button` class is used to position the button under the calendar. It's toggling `cdk-visually-hidden` above to show/hide the button.